bookserver/main.py

```diff
@@ -135,9 +137,29 @@ def level2_validation_handler(request: Request, exc: ValidationError):
     secondary validation when populating our xxx_answers tables
     this catches those and returns a 422
     """
-    # rslogger.debug(exc.json)
+    rslogger.error(exc)
 
     return JSONResponse(
         status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
         content=jsonable_encoder({"detail": exc.errors()}),
     )
+
+
+@app.exception_handler(Exception)
```

Comment:
Very nice! Would you mind adding a link to https://fastapi.tiangolo.com/tutorial/handling-errors/#install-custom-exception-handlers? Or whatever FastAPI docs seem relevant?